tests/ddl_no_csc2.test/t11_cascade_drop.expected

```diff
@@ -0,0 +1,24 @@
+(name='ddl_cascade_drop', description='On DROP, also drop the dependent keys/constraints. (Default: 1)', type='BOOLEAN', value='ON', read_only='Y')
```

Comment:
We also have a similar test here : https://github.com/bloomberg/comdb2/blob/c6b3b2ef685c24ca0a121bf28037da1be296011f/tests/ddl_no_csc2.test/t01_alter_table.sql#L119

Reply:
oh, nice. no harm having it twice i guess.
Thanks